raft.c

```diff
@@ -437,6 +471,10 @@ static int raftApplyLog(raft_server_t *raft, void *user_data, raft_entry_t *entr
                 LOG_DEBUG("Removing this node from the cluster");
                 return RAFT_ERR_SHUTDOWN;
             }
+
+            if (raft_is_leader(raft)) {
+                raftSendNodeRemoved(raft_get_node(raft, req->id));
+            }
```

Comment:
do we want to avoid sending if if we are removing self?

Reply:
has self check just above this line :)

Reply:
ahh, missed it.